--- a/cage.c
+++ b/cage.c
@@ -47,6 +47,7 @@
 #include <wlr/types/wlr_data_device.h>
 #include <wlr/types/wlr_export_dmabuf_v1.h>
 #include <wlr/types/wlr_foreign_toplevel_management_v1.h>
+#include <wlr/types/wlr_fractional_scale_v1.h>
 #include <wlr/types/wlr_gamma_control_v1.h>
 #include <wlr/types/wlr_idle_inhibit_v1.h>
 #include <wlr/types/wlr_idle_notify_v1.h>
@@ -367,6 +368,10 @@ main(int argc, char *argv[])
 	if (!wlr_single_pixel_buffer_manager_v1_create(server.wl_display)) {
 		wlr_log(WLR_ERROR, "Unable to create the single pixel buffer manager");
 		ret = 1;
 		goto end;
 	}
+	if (!wlr_fractional_scale_manager_v1_create(server.wl_display, 1)) {
+		wlr_log(WLR_ERROR, "Unable to create the fractional scale manager");
+		ret = 1;
+		goto end;
+	}
 	if (!wlr_xdg_output_manager_v1_create(server.wl_display, server.output_layout)) {
